app/models/article.rb

```diff
@@ -0,0 +1,14 @@
+class Article < ApplicationRecord
+  extend Enumerize
+
+  paginates_per 5
+
+  AVALIABLE_STATUSES = ["public", "draft", "archive"].freeze
+
+  belongs_to :user
+
+  validates :title, :content, presence: true, length: { minimum: 3 }
+  validates :status, presence: true
+
+  enumerize :status, in: AVALIABLE_STATUSES , scope: :shallow
```

Comment:
enumerize also provides default option, you can use it
```suggestion
  enumerize :status, in: AVALIABLE_STATUSES , scope: :shallow, default: :draft
```

Reply:
Are we need to use default option in the database and in ActiveRecord too?